Library/Homebrew/cmd/audit.rb

```diff
@@ -668,6 +668,22 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    Pathname.glob("#{formula.prefix}/**/*").each do |file|
```

Comment:
... which should help the pathological case where there are hundreds of formula installed.